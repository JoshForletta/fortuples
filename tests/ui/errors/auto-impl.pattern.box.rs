#[fortuples::auto_impl]
trait Test {
    fn test(box a: Box<i32>) {
        println!("{}", a);
    }
}

fn main() {}
