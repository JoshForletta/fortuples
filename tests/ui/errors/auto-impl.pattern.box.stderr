error: `auto_impl` supports only ident of wildcard arguments
 --> tests/ui/errors/auto-impl.pattern.box.rs:3:13
  |
3 |     fn test(box a: Box<i32>) {
  |             ^^^
